### DISCUSSION TITLE
Elementary OS Wi-Fi connection with raspberry pi

### DISCUSSION THREAD

**Discussion by aXdey65i** [elementary/triage#614] [Q&A]
I'm setting up an elementary OS on the raspberry pi, but the only option is Ethernet. Please tell me how to add Wi-Fi.

**Comment by ryonakano** [comment]
We need to know which version of elementary OS and which model of Raspberry Pi you're using first. Are you trying elementary OS 8 Daily on Raspberry Pi 4?

**Reply by aXdey65i** [reply]
8で試しています。

**Reply by ryonakano** [reply]
Could you share your the result of `ip a` and the screenshot of Network pane in System Settings? I'd like to know if the network interface for WLAN is detected by the kernel and just the problem of configuration in NetworkManager.

**Reply by aXdey65i** [reply]
Sorry for being late! I can't find Wi-Fi like this. Is there a solution?
![IMG_1264](https://github.com/user-attachments/assets/daa71a27-1531-4e28-936f-dad747605cc9)

**Reply by ryonakano** [reply]
Thank you for sharing the screenshot and I'm very sorry for my late reply.

Looking at the screenshot your Wi-Fi device does not seem to be detected. An entry with titled as "Wireless" should be displayed in the devices list in this screen if it's detected correctly.

Could you share the result of `ip a` and `lspci` command?
